bin/hyperspy

```diff
@@ -145,6 +145,13 @@ for f in to_copy:
             destination, os.path.split(f)[-1])) or \
             args.overwrite_profile:
         shutil.copy(f, destination)
+if ipy_version >= LooseVersion('3.0.0'):
+    additional_dest = os.path.join(destination, 'startup')
+    if not os.path.isdir(additional_dest):
+        os.makedirs(additional_dest)
+    if not os.path.isfile(os.path.join(additional_dest, 'hyperspy_magics.py')) or \
```

Comment:
If in  a new release we change `hyperspy_magics.py`, the copy in the user's profile won't be updated unless he asks to overwrite the profile. Couldn't the content for `hyperspy_magics.py` be actually imported from the `hyperspy` module itself in order to workaround this?